### ISSUE TITLE
Examples?

### ISSUE THREAD

**Issue by NickPea** [dotnet/docs#15951]
Probably put some use examples in as well to keep this page consistent with the rest of the documentation.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fbdace9d-6fa0-0aec-25b4-df6f414001d7
* Version Independent ID: 77eff122-b9a5-2b1f-3a68-df302cb46194
* Content: [Set Operations (C#)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/set-operations#feedback)
* Content Source: [docs/csharp/programming-guide/concepts/linq/set-operations.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/concepts/linq/set-operations.md)
* Product: **dotnet-csharp**
* Technology: **csharp-linq**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by SuperNova911** [comment]
I think I can make some sample for this
It looks like code is managed in [here](https://github.com/dotnet/samples/tree/master/snippets/csharp/VS_Snippets_VBCSharp), but how can I link with *Set Operations (C#)* article??

Should I just make duplicate code for article and dotnet/sample?

**Comment by BillWagner** [comment]
Thanks for writing this @NickPea We should add examples here. I've added this to our backlog, and added the "up for grabs" label.

@SuperNova911 Thanks for asking about contributing to fix this issue. I've got a few thoughts on the samples:

- These could be good candidates for the [interactive experience](https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md#the-c-interactive-experience).
- Most of our samples in this area were migrated from an internal system, and don't match how we're creating samples now. I'd recommend adding samples in a new LINQ folder under the [snippets/csharp/(https://github.com/dotnet/samples/tree/master/snippets/csharp) folder in the samples repository.

Fixing this would be two PRs, one in the dotnet/samples repo, and one in this repo to include the new examples.

@ - mention me with any questions.
